File: Java/Card.java
package Java;

class Card extends Payment {
    String numberCard;
    String dateCard;
    String cvv;

    public Card(String numberCard, String dateCard, String cvv) {
        this.numberCard = numberCard;
        this.dateCard = dateCard;
        this.cvv = cvv;
    }
}

File: PHP/card.php
<?php
require_once('Payment.php');

class Card extends Payment{
    public $numberCard;
    public $dateCard;
    public $cvv;

    public function __construct($numberCard, $dateCard, $cvv)
    {
        $this->$numberCard = $numberCard;
        $this->$dateCard = $dateCard;
        $this->$cvv = $cvv;
    }
}

?>